library/src/main/res/values/library_android_textview_linkbuilder_strings.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<resources>
+    <string name="define_android_textview_linkbuilder"></string>
```

Comment:
please add `_int` to the define definition. 
without `_int` those libs will always be displayed. 


It should look like this: https://github.com/mikepenz/AboutLibraries/blob/develop/library/src/main/res/values/library_activeandroid_strings.xml#L4

Please also do this for the other 2 libs